OpenRA.Mods.RA/OpenRA.Mods.RA.csproj

```diff
@@ -132,22 +131,14 @@
     <Compile Include="Traits\MadTank.cs" />
     <Compile Include="Traits\Mine.cs" />
     <Compile Include="Traits\Minelayer.cs" />
-    <Compile Include="Orders\PlaceBuildingOrderGenerator.cs" />
-    <Compile Include="Orders\GlobalButtonOrderGenerator.cs" />
     <Compile Include="ParaDrop.cs" />
-    <Compile Include="Player\ClassicProductionQueue.cs" />
-    <Compile Include="Player\PlaceBuilding.cs" />
-    <Compile Include="Player\ProductionQueue.cs" />
     <Compile Include="Traits\PortableChrono.cs" />
     <Compile Include="Scripting\Properties\GuardProperties.cs" />
     <Compile Include="Widgets\Logic\TabCompletionLogic.cs" />
-    <Compile Include="Production.cs" />
-    <Compile Include="ProductionBar.cs" />
     <Compile Include="Traits\Render\RenderJammerCircle.cs" />
-    <Compile Include="Render\RenderBuildingWarFactory.cs" />
-    <Compile Include="Render\RenderHarvester.cs" />
-    <Compile Include="Render\RenderDisguise.cs" />
-    <Compile Include="Render\RenderLandingCraft.cs" />
+    <Compile Include="Traits\Render\RenderHarvester.cs" />
```

Comment:
Yes, but this requires the ```Harvester``` trait, and you said I shouldn't touch that stuff yet, so I just moved it inside Mods.RA to get rid of the old ```Render``` folder ;)